src/SonarLint.VSSpecificAssemblies.props

```diff
@@ -46,17 +46,6 @@
     <VsTargetMajorMinorVersion Condition="$(VersionSpecificSuffix) == '2022' ">17.0</VsTargetMajorMinorVersion>
   </PropertyGroup>
 
-  <!-- ******************************************************************************* -->
-  <!-- Legacy VS static analysis ruleset -->
-  <!-- ******************************************************************************* -->
-
-  <ItemGroup Label="Legacy VS CodeAnalysis" Condition=" $(RequiresRuleset)=='true' " >
-    <Reference Include="Microsoft.VisualStudio.CodeAnalysis">
-      <HintPath>$(ThirdPartyPath)\Microsoft.VisualStudio.CodeAnalysis.dll</HintPath>
-      <Private>$(CopyVSAssembliesToOutput)</Private>
-    </Reference>
-  </ItemGroup>
```

Comment:
This is what actually added the correct assembly reference to the project. No longer required.

(if you expand the file you'll see there are a couple of other third-party binaries that are still handled the same way).